th/2007/winter.json

```diff
@@ -1,76 +1,52 @@
 {
-  "name": "ชื่อ",
-  "group": "แฟนซับ",
-  "message": "ไม่ทราบข้อมูล",
-  "lbl_tv": "TV",
-  "lbl_ova": "OVA/ONA/SP",
-  "lbl_movie": "Movie",
-  "tv": [
-    {
-      "name": "Minami-ke Okawari",
-      "image": "https://myanimelist.cdn-dena.com/images/anime/10/73556.jpg?s=acda84937850229c72e1772d3ba46d74",
-      "group": [
-        {
-          "status": "release",
-          "detail": [
-            {
-              "name": "Endless-FS"
-            },
-            {
-              "name": "OxygenASz-FS"
-            }
-          ]
-        },
-        {
-          "status": "release",
-          "detail": [
-            {
-              "name": "OxygenASz-FS"
-            }
-          ]
-        }
-      ]
-    },
-    {
-      "name": "True Tears",
-      "image": "https://myanimelist.cdn-dena.com/images/anime/8/18578.jpg?s=a8c1b1add79da91b24b63505ea34aa78",
-      "group": [
-        {
-          "status": "release",
-          "detail": [
-            {
-              "name": "Thai-Fandub"
-            }
-          ]
-        },
-        {
-          "status": "release",
-          "detail": [
-            {
-              "name": "x105sub"
-            }
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Zoku Sayonara Zetsubou Sensei",
-      "image": "https://myanimelist.cdn-dena.com/images/anime/13/75517.jpg?s=5782a5ecb7e8393d1d632db59c8a4f4e",
-      "group": [
-        {
-          "status": "release",
-          "detail": [
-            {
-              "name": "Bleachsoul"
-            },
-            {
-              "name": "Elaborate - FS"
-            }
-          ]
-        }
-      ]
-    }
-  ],
-  "ova": [],
-  "movie": []
-}
\ No newline at end of file
+	"name": "ชื่อ",
+	"group": "แฟนซับ",
+	"message": "ไม่ทราบข้อมูล",
+	"lbl_tv": "TV",
+	"lbl_ova": "OVA/ONA/SP",
+	"lbl_movie": "Movie",
+	"tv": [{
+		"name": "Minami-ke Okawari",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/10/73556.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "Endless-FS"
+			}, {
+				"name": "OxygenASz-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "OxygenASz-FS"
```

Comment:
from menome data dump OxygenASz-FS had release Minami-ke Okawari EP.1 without Endless-FS
But in EP.2 (and later)  OxygenASz-FS release with Endless-FS.

So i think it must be OxygenASz-FS & Endless-FS.